### DISCUSSION TITLE
#Hacktoberfest2020 PR's

### DISCUSSION THREAD

**Discussion by shivambhat** [EddieHubCommunity/support#314] [Q&A]

**Comment by eddiejaoude** [comment]
Please do not write in bold, there is no need to shout

Do you want to share some links to these PRs so we are not guessing?

**Reply by shivambhat** [reply]
Hello  😊 @eddiejaoude 

### Sorry ,but I meant no disrespect. I was just trying to highlight my query. That's all about it! 😅
Sure I will share the links. Do you want me to share it here?

**Reply by shivambhat** [reply]
Hello 😊 @eddiejaoude
 
 Sorry ,but I meant no disrespect. I was just trying to highlight my query. That's all about it! 😅
 Sure I will share the links. Do you want me to share it here?

**Reply by eddiejaoude** [reply]
Yes you can share them here.

**Reply by eddiejaoude** [reply]
PS. you can update your messages so they are less "shouting" 😉

**Reply by shivambhat** [reply]
> PS. you can update your messages so they are less "shouting" 😉

@eddiejaoude  Hahaha understood ...

**Comment by shivambhat** [comment]
Links for my PR's which still hasn't been accepted:
https://github.com/bhaveshlohana/HacktoberFest2020-Contributions/pull/1172
https://github.com/Rishikesh-12/HacktoberFest2020/pull/656
https://github.com/Rishikesh-12/HacktoberFest2020/pull/654/files
https://github.com/bhaveshlohana/HacktoberFest2020-Contributions/pull/1172
https://github.com/swapnanildutta/Hackerrank-Codes/pull/108
https://github.com/bajajvinamr/HacktoberFest2020/pull/479

Please check & let me know what can be done to get these worked on.

PS:Now  2/9  PR's got approved.

**Reply by eddiejaoude** [reply]
Can you share the hacktoberfest dashboard of your status?

**Reply by eddiejaoude** [reply]
Most of those PRs look like practice Hacktoberfest repos - so they might get excluded

**Reply by shivambhat** [reply]
@eddiejaoude 
Dashboard link: https://hacktoberfest.digitalocean.com/profile

3 of my PR's are in review, Please do check and tell me if  there PR's are good to go?

**Reply by shivambhat** [reply]
@eddiejaoude   Any update?

**Reply by eddiejaoude** [reply]
I took a look but the link you shared is a generic link to the profile of the person who is logged in not your profile. If you can not get a public link, take a screenshot

**Reply by shivambhat** [reply]
![opensource](https://user-images.githubusercontent.com/18063625/96154106-54db4580-0ec3-11eb-99d2-7c79ec1febba.jpg)

@eddiejaoude

**Reply by RemcoHalman** [reply]
@shivambhat The exclamation mark means: 
```bash
Ineligible Repository
Your PR was submitted to a repository that is not participating in Hacktoberfest. Maintainers of the repository can add the "hacktoberfest" topic to their repository if they wish to participate. Alternatively, an individual PR can be opted-in with a maintainer adding the "hacktoberfest-accepted" label to the PR.
```

so you could ask the maintainer of the repo to add `hacktoberfest-accepted`
